source/databricks/calculation_engine/package/databases/table_column_names.py

```diff
@@ -30,6 +35,7 @@ class TableColumnNames:
     calculation_execution_time_start = "calculation_execution_time_start"
     calculation_succeeded_time = "calculation_succeeded_time"
     calculation_id = "calculation_id"
+    calculation_result_id = "calculation_result_id"
```

Comment:
I believe this should be:
```suggestion
    result_id = "result_id"
```

Reply:
can i add a todo on it and fix in another PR, I dont want to change schemas or data in this one

Reply:
Yes, good idea to add a todo. But can you update the field name and leave the string value and add a comment about value. Just so that we don't use the bad name in all the code.

Reply:
there is already a result_id